packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -502,6 +502,15 @@ class RawKeyboard {
     if (event == null) {
       return;
     }
+    if (event.data is RawKeyEventDataMacOs && event.logicalKey == LogicalKeyboardKey.fn) {
```

Comment:
Does `RawKeyEventDataMacOs` only apply to macOS, or is it the entire Apple platform (i.e. iOS, iPadOS, etc)?

Reply:
It only applies to macOS.  We don't have any keyboard support on iOS at all (yet).